### PR TITLE
[HOTFIX] fix: add note about lb error behavior and timeout config

### PR DESCRIPTION
**Pull request by bru-andrade**
### Changes

- Added note to timeout to contact support
- Added behavior to balancing method

### Additional links

https://aziontech.slack.com/archives/C05DRB16170/p1695644703084199

[EDU-3000]: https://aziontech.atlassian.net/browse/EDU-3000?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/en/pages/main-menu/reference/build/edge-application/load-balancer.mdx [hunk 1]
```diff
@@ -28,6 +28,12 @@ With Load Balancer, you can:
 
 The balancing method defines how the load will be distributed among your origins.
 
+:::note
+**Load Balancer** will query the origins according to the balancing method and weight they are assigned. If an origin returns a `4xx` or a `5xx` error, or takes longer to respond than the [timeouts](/en/documentation/products/edge-application/origins/#timeouts) defined in the origin, the Load Balancer will override the balancing method and weight to query the remaining origins before returning an error to your users.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/main-menu/reference/build/edge-application/load-balancer.mdx:32]:
```suggestion
**Load Balancer** will query the origins according to the balancing method and weight they're assigned. If an origin returns a `4xx` or a `5xx` error, or takes longer to respond than the [timeouts](/en/documentation/products/edge-application/origins/#timeouts) defined in the origin, the Load Balancer will override the balancing method and weight to query the remaining origins before returning an error to your users.
```

Comment (MarianaAguilera) [reply src/content/docs/en/pages/main-menu/reference/build/edge-application/load-balancer.mdx:32]: 'Load Balancer" or "the load balancer"?

File: src/content/docs/en/pages/main-menu/reference/build/edge-application/load-balancer.mdx [hunk 2]
```diff
@@ -28,6 +28,12 @@ With Load Balancer, you can:
 
 The balancing method defines how the load will be distributed among your origins.
 
+:::note
+**Load Balancer** will query the origins according to the balancing method and weight they're assigned. If an origin returns a `4xx` or a `5xx` error, or takes longer to respond than the [timeouts](/en/documentation/products/edge-application/origins/#timeouts) defined in the origin, the Load Balancer will override the balancing method and weight to query the remaining origins before returning an error to your users.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/main-menu/reference/build/edge-application/load-balancer.mdx:32]: "Load Balancer"ou "the load balancer"?

File: src/content/docs/pt-br/pages/menu-principal/referencia/build/edge-application/load-balancer.mdx [hunk 3]
```diff
@@ -28,6 +28,12 @@ Com o Load Balancer, você pode:
 
 O método de balanceamento define como a carga será distribuída entre suas origens.
 
+:::note
+O **Load Balancer** irá consultar as origens de acordo com o método de balanceamento e o peso atribuído a elas. Se uma origem retornar um erro `4xx` ou `5xx`, ou demorar mais temp para responder do que os [timeouts](/pt-br/documentacao/produtos/edge-application/origins/#timeouts) definidos na origem, o Load Balancer irá sobrepor o método de balanceamento e peso e consultar as origens restantes antes de retornar um erro aos seus usuários.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/referencia/build/edge-application/load-balancer.mdx:32]:
```suggestion
O **Load Balancer** irá consultar as origens de acordo com o método de balanceamento e o peso atribuído a elas. Se uma origem retornar um erro `4xx` ou `5xx`, ou demorar mais tempo para responder do que os [timeouts](/pt-br/documentacao/produtos/edge-application/origins/#timeouts) definidos na origem, o Load Balancer irá sobrepor o método de balanceamento e peso e consultar as origens restantes antes de retornar um erro aos seus usuários.
```

**Review by MarianaAguilera — APPROVED** [review]

File: src/content/docs/en/pages/main-menu/reference/build/edge-application/load-balancer.mdx (hunk 2 above)
Comment (bru-andrade) [reply src/content/docs/en/pages/main-menu/reference/build/edge-application/load-balancer.mdx:32]: O produto mesmo, acho que não existe uma entidade load balancer, como é um origin type. :/

File: src/content/docs/en/pages/main-menu/reference/build/edge-application/load-balancer.mdx [hunk 4]
```diff
@@ -28,6 +28,12 @@ With Load Balancer, you can:
...
+
+To customize this behavior, [contact the Support team](https://tickets.azion.com/).
```
Comment (hannah-sk) [inline src/content/docs/en/pages/main-menu/reference/build/edge-application/load-balancer.mdx:34]:
```suggestion
To customize this behavior, [contact the support team](https://tickets.azion.com/).
```

**Review by hannah-sk — APPROVED** [review]

**Pull request merged**